site/assets/scss/_component-examples.scss

```diff
@@ -2,12 +2,15 @@
 // Docs examples
 //
 
-.bd-example-snippet {
+.bd-code-snippet {
+  margin-right: ($bd-gutter-x * -.5);
```

Comment:
Maybe it's because I'm not used to it but TBH removing the x margin feels off to me.

In these cases, the left alignment disturbs me:
<img width="878" alt="Screenshot 2023-04-26 at 09 40 39" src="https://user-images.githubusercontent.com/17381666/234504655-a505d3d4-0558-4780-bba7-c5bcf052f448.png">

<img width="182" alt="Screenshot 2023-04-26 at 09 40 11" src="https://user-images.githubusercontent.com/17381666/234504671-16bbd736-38c1-4bb3-aaed-3342aaa44b61.png">

<img width="128" alt="Screenshot 2023-04-26 at 09 45 51" src="https://user-images.githubusercontent.com/17381666/234505814-255c2522-2549-4fb0-9387-7629815f4d2f.png">


In this case, the right border is really close to the violet active left border:
<img width="273" alt="Screenshot 2023-04-26 at 09 40 31" src="https://user-images.githubusercontent.com/17381666/234504667-ab0d59f5-86c4-41e7-b501-fd3590a91f5b.png">


With the desktop and tablet breakpoints, when there are 2 columns, having 2 code snippets touching each other is also disturbing:

<img width="1315" alt="Screenshot 2023-04-26 at 09 39 32" src="https://user-images.githubusercontent.com/17381666/234504686-edaeb959-f6d2-4d89-a043-f867d0cbd75a.png">
<img width="1038" alt="Screenshot 2023-04-26 at 09 39 23" src="https://user-images.githubusercontent.com/17381666/234504698-951ff13f-d3b7-4071-a9cf-c8b4ca0aa9da.png">